### ISSUE TITLE
SpriteAnimPlayOpts seem to be ignored?

### ISSUE THREAD

**Issue by JoshuaGrams** [replit/kaboom#653]
### Version

v2000.2.9

### What browsers are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What happened?

It uses the speed from when the sprite was created.

### What's the expected behavior?

Use the speed in the options passed to `mySprite.play('animName', {speed: 2})`

### Minimum reproducable code

```shell
// NOTE: your issue template says "Minumum reproducable code" (should be "reproducible")

kaboom()
loadSprite('seal', 'seal.png', {
	sliceY: 2,
	anims: { run: {from: 0, to: 1, loop: true} }
})
const seal = add([sprite('seal')])
seal.play('run', {speed: 2})
```

**Comment by lajbel** [comment]
It's true, probably isnt implemented or there is a bug. For the moment, you can define the speed in the `loadSprite` anim. Thanks

**Comment by slmjkdbtl** [comment]
Just published [v2000.2.10](https://github.com/replit/kaboom/commit/383c07340606078ee64e9f57ca1fab88b46c4ac2) to fix this!
